src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -16,154 +14,108 @@ import {
 
 import { User } from "types/User";
 
-// @debt remove styled-components in favour of using our standard scss patterns
-import * as S from "./UserProfilePicture.styles";
+import { UserReactions } from "./UserReactions";
+
 import "./UserProfilePicture.scss";
 
+const randomAvatarUrl = (id: string) =>
```

Comment:
generate/getRandomAvatarUrl?

Reply:
Agreed. This naming was legacy, but didn't sit right with me, just hadn't thought of something better.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/5efb685f8665173e76d562c8896cae5b5d31cb12